client/engine/chainservice/eth_chainservice.go

```diff
@@ -124,6 +123,7 @@ func (ecs *EthChainService) listenForLogEvents(na *NitroAdjudicator.NitroAdjudic
 					panic(err)
 				}
 
+				// TODO set Holdings based on chain event
```

Comment:
Shall we make this a full-blown issue?

Reply:
That would be good, though this work is currently in progress.